clang/lib/Sema/SemaExpr.cpp

```diff
@@ -18146,6 +18148,16 @@ void Sema::MarkFunctionReferenced(SourceLocation Loc, FunctionDecl *Func,
                 std::make_pair(Func, PointOfInstantiation));
             // Notify the consumer that a function was implicitly instantiated.
             Consumer.HandleCXXImplicitFunctionInstantiation(Func);
+
+            if (llvm::isTimeTraceVerbose()) {
```

Comment:
nit: move this closer to the `push_back` since they are related

Reply:
Done.